src/shipit_static_analysis/shipit_static_analysis/lint.py

```diff
@@ -155,14 +154,14 @@ def find_issues(self, path):
             logger.warn('Invalid json output', path=path, lines=lines)
             raise
 
-        full_path = os.path.join(self.repo_dir, path)
+        full_path = os.path.join(settings.repo_dir, path)
         if full_path not in payload and path not in payload:
             logger.warn('Missing path in linter output', path=path)
             return
 
         # Mozlint uses both full & relative path to index issues
         return [
-            MozLintIssue(self.repo_dir, path, **issue)
+            MozLintIssue(path, **issue)
             for p in (path, full_path)
-            for issue in payload.get(p, [])
+            for issue in payload[full_path]
```

Comment:
Why this change? I remember you saw mozlint issues appearing either with full paths or relative paths.